src/components/ComponentConfiguration.stories.tsx

```diff
@@ -3059,3 +3059,55 @@ export const SoftRequiredErrors: Story = {
     });
   },
 };
+
+export const Partners: Story = {
+  render: Template,
+  name: 'type: partners',
+
+  args: {
+    component: {
+      id: 'wekruya',
+      type: 'partners',
+      key: 'partners',
+      label: 'Partners',
+      tooltip: 'An example for the tooltip',
+      description: 'A description for the Partners component',
+      defaultValue: [
```

Comment:
does it make sense to write stories with `defaultValue` when we don't allow this to be set?